drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/client_involvement.rb

```diff
@@ -0,0 +1,87 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis
+  class ClientInvolvement
+    attr_accessor :client_ids
+    attr_accessor :end_date
+    attr_accessor :mci_ids
+    attr_accessor :start_date
+    attr_accessor :status_message
+
+    def initialize(params)
+      self.end_date = params[:end_date]
+      self.start_date = params[:start_date]
+      self.mci_ids = params[:mci_ids]
+    end
+
+    def validate_request!
+      message = []
+
+      message << 'mci_ids not provided.' if mci_ids.blank?
+      message << 'start_date not provided.' if start_date.blank?
+      message << 'end_date not provided.' if end_date.blank?
+
+      begin
+        self.start_date = Date.strptime(start_date, '%Y-%m-%d') if start_date.present?
+      rescue Date::Error
+        message << 'start_date was not formatted correctly.'
+      end
+
+      begin
+        self.end_date = Date.strptime(end_date, '%Y-%m-%d') if end_date.present?
+      rescue Date::Error
+        message << 'end_date was not formatted correctly.'
+      end
+
+      # FIXME: Fetch the right way once it's settled
```

Comment:
@yakloinsteak heads up the namespace change has been merged in now: https://github.com/greenriver/hmis-warehouse/pull/3009